src/index.ts

```diff
@@ -30,14 +30,18 @@ const getValidationContext = (validation: t.ValidationError) =>
   // https://github.com/gcanti/fp-ts/pull/544/files
   validation.context as t.ContextEntry[];
 
+export const TYPE_MAX_LEN = 160; // Two lines of 80-col text
+const truncateType = (type: string): string =>
+  type.length > TYPE_MAX_LEN ? `${type.slice(0, TYPE_MAX_LEN - 3)}...` : type;
```

Comment:
An alternative is to do the `...` in the middle and show both the beginning and end, to make it feel more "complete"

Reply:
That would look better I think

Reply:
Yeah, it would look better.